Form-Controls/index.html

```diff
@@ -9,19 +9,48 @@
   </head>
   <body>
     <header>
-      <h1>Product Pick</h1>
+      <h1>T-shirt Select Form</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <h3>Customer Information</h3>
+        <div><label for="fullname">Full Name:</label>
+          <input type="text" id="fullname" name="fullname" placeholder="Your full name" required></div>
+         
+          <br>
+        
+        <div><label for="email">Email:</label>
+          <input type="email" id="email" name="email" placeholder="Your email" required></div>
+        
+          <br>
+        <h3>T-shirt Options</h3>
+        <div><label for="colour">T-shirt Colour:</label>
+          <select id="colour" required>
+            <option value="" disabled selected> Please select a colour</option>
+            <option value="Red">Red</option>
+            <option value="White">White</option>
+            <option value="Black">Black</option>
+          </select>
+        </div>
+
+          <br>
+
+          <div><label for="size">T-shirt Size:</label>
+            <select id="size" required>
+              <option value="" disabled selected> Please select a colour</option>
+              <option value="Red">X-SMALL</option>
+              <option value="White">SMALL</option>
+              <option value="Black">Medium</option>
+              <option value="Black">Large</option>
+              <option value="Black">X-Large</option>
+              <option value="Black">XX-Large</option>
+            </select>
+          </div>
       </form>
     </main>
     <footer>
       <!-- change to your name-->
```

Comment:
It's a good idea to remove comments that don't add value to the code. Valuable comments are those that explain complex business logic or there is a need to clarify the intent of a code implementation.